src/client/components/HomePage/HomePage.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
+import style from './style.css';
+
+const events = [
+    {
+        date: '28 X 2017',
+        place: 'Gruba Blues',
+        city: 'Chorzów'
+    },
+    {
+        date: '12 XII 2017',
+        place: 'Dni Piekar Śląskich',
+        city: 'Piekary Śląskie'
+    },
+    {
+        date: '03 III 2017',
+        place: 'Festyn Sportowy',
+        city: 'Park Murckowski'
+    }
+];
 
 export const HomePage = () => (
-    <div>Strona główna</div>
+    <section className={style.section}>
+        <h2 className={style.title}>Gdzie gramy ?</h2>
+        <div className={style.events}>
+            28 X 2017 ---- Gruba Blues ---- Chorzów<br/>
```

Comment:
Skoro masz dostęp do zmiennej *events*, to możesz normalnie odnieść się do niej tutaj w JSXie używając klamerek:
```jsx
<div className={style.events}>
    {events.map((event) => (
        <span>{event.date} ---- {event.place} ---- {event.city}</span>
    ))}
</div>
```
W tym wypadku tablica metoda *.map()* na dowolnej tablicy (a taką jest *events*) "mapuje" każdy element tablicy na inny. Innymi słowy tworzy i zwraca nową tablicę na podstawie starej - tutaj dla każdego obiektu wewnątrz *events* zwraca element HTML span do którego wstawia informacje o danym evencie (i tak dla każdego elementu tablicy).

Reply:
Dzięki :) Własnie tego potrzebowałem. Nie wiedziałem jak umieścić to w html. Nie wiem czy widziałeś mój kom "Dodałem consta o nazwie events. Pomyślałem że w wyświetlaniu wydarzeń można by zrobić podobnie jak z menu. Czyli mamy sobie zbiór który automatycznie generuje nam listę wydarzeń. Nie wiem czy dobrze kombinuje ? Jeśli tak to nie wiem teraz jak wyświetlić to w html"   Spróbuję to w taki sposób zamieścić.